code/modules/client/client_defines.dm

```diff
@@ -156,3 +156,19 @@
 	var/parallax_movedir = 0
 	var/parallax_layers_max = 4
 	var/parallax_animate_timer
+
+	/**
+	 * Assoc list with all the active maps - when a screen obj is added to
+	 * a map, it's put in here as well.
+	 *
+	 * Format: list(<mapname> = list(/obj/screen))
+	 */
+	var/list/screen_maps = list()
+
+	// List of all asset filenames sent to this client by the asset cache, along with their assoicated md5s
+	var/list/sent_assets = list()
+	/// List of all completed blocking send jobs awaiting acknowledgement by send_asset
+	var/list/completed_asset_jobs = list()
+	/// Last asset send job id.
+	var/last_asset_job = 0
+	var/last_completed_asset_job = 0

```

Comment:
docs

Reply:
Honestly I'm just moving the variables around, not planning on adding docs for them since I'm not changing their functionality. It's not relevant for this PR, the logic stays the same.

If you suggest changes I can apply them, though.

Reply:
@Rohesie idk just like /// time of last completed asset job or whatever the var tracks

Reply:
suggest those chances, I'll accept them